el/pagination.php

```diff
@@ -14,7 +14,6 @@
     */
 
     'previous' => '&laquo; Προηγούμενη',
-
```

Comment:
hehe I thought it would be better if it was exactly the same. So, do you want me to edit and commit this again?